subway_stations.rb

```diff
@@ -0,0 +1,58 @@
+require "byebug"
+#binding.byebug
+
+# Ask our user where they are
+
+# puts " What station are you on?"
+# origin_station = gets().chomp()
```

Comment:
What was your reason for commenting out all your input code. Were you trying to debug? What was the issue?